pkg/limayaml/limayaml.go

```diff
@@ -83,6 +84,13 @@ var (
 	VMTypes    = []VMType{QEMU, VZ, WSL2}
 )
 
+type User struct {
+	Name    *string `yaml:"name,omitempty" json:"name,omitempty" jsonschema:"nullable"`
+	Comment *string `yaml:"comment,omitempty" json:"comment,omitempty" jsonschema:"nullable"`
+	Home    *string `yaml:"home,omitempty" json:"home,omitempty" jsonschema:"nullable"`
+	UID     *string `yaml:"uid,omitempty" json:"uid,omitempty" jsonschema:"nullable"`
```

Comment:
Why not `*int`?

Reply:
No particular reason, I can change it.

Reply:
@AkihiroSuda Can we leave this for a later refactoring PR? I would also want to combine this type with https://github.com/lima-vm/lima/blob/9fe06aa4cc7ac7d1d6b72787301753cc324ae4d8/pkg/osutil/user.go#L17-L24

But I feel like the current PR is already big enough.